spec/fixtures/modules/facts/plans/emit.pp

```diff
@@ -0,0 +1,4 @@
+plan facts::emit(String $host) {
+  $target = get_targets($host)[0]
+  return "Facts for ${host}: ${facts($target)}"
```

Comment:
😮 there is a return in puppet?

Reply:
Yup, for functions. PUP-8608 intends to change plans to rely on it for returning anything.